storybook/pages/StatusMessagePage.qml

```diff
@@ -40,9 +40,118 @@ SplitView {
                 {
                     emoji: "🕵️‍♀",
                     didIReactWithThisEmoji: false,
-                    numberOfReactions: 2,
+                    numberOfReactions: 232,
                     jsonArrayOfUsersReactedWithThisEmoji: "[\"Bob\", \"John\"]"
+                },
```

Comment:
Maybe we could create a specific file with the reactions model so that it's reusable and doesn't add a lot of noise to this specific page?